test/test.cc

```diff
@@ -385,8 +385,14 @@ boost::property_tree::ptree make_config(const std::string& path_prefix,
     }
   )";
 
+  // we should use absolute paths so the binaries work regardless of CWD
+  std::string absolute;
+  if (path_prefix.find("test/") == 0) {
+    absolute = VALHALLA_BUILD_DIR;
+  }
+
   // force the paths to be different
-  boost::replace_all(defaults, "%%", path_prefix);
+  boost::replace_all(defaults, "%%", absolute + path_prefix);
```

Comment:
this is the important part, we detect if your data prefix doesnt have the absolute path on and add it if not. this lets the binaries be run from any working directory (which can be useful)

Reply:
also note that `gurka::buildtiles` calls into this function so its covered too